### ISSUE TITLE
feat(chore): provide JSON schema for config file

### ISSUE THREAD

**Issue by dzmitry-lahoda** [paritytech/zombienet#1331]
### Is your feature request related to a problem? Please describe.

For ZN config, please provide JSON spec so it is not procedural hard to follow, but well documented in JSON schema, so can validate it and fail on bad paraters.

### Describe the solution you'd like

JSON spec -> TS code to handle it, not TS code

**Comment by pepoviola** [comment]
related to https://github.com/paritytech/zombienet/issues/662.
